opennlp-tools/src/main/java/opennlp/tools/cmdline/GenerateManualTool.java

```diff
@@ -153,8 +154,9 @@ private static void appendArgumentTable(
             "<entry>Optional</entry><entry>Description</entry></row></thead>\n");
     sb.append("<tbody>\n");
 
-    for (String format : formatArguments.keySet()) {
-      List<Argument> arguments = formatArguments.get(format);
+    for (Entry<String, List<Argument>> entry : formatArguments.entrySet()) {
```

Comment:
Less performatic. Calling Map#keySet() you will start an iterator to access the map elements, and then for each element you access the map again to retrieve the element with Map#get(key).

What FindBugs suggests is that instead we use Map#entrySet(), which will iterate the map once, retrieving both key+value. So it should normally be faster to iterate the map this way.